### PR TITLE
Add initial business process handler for consumer move in

### PR DESCRIPTION
**Pull request by kft**
## Description
This PR adds an initial implementation for accepting and registering consumer move in business processes:

Changes/addtions:

- Business request handler
- Integration test that verifies publication of `MoveInRequestAccepted` actor message.
- A couple of refactorings to `TestHost`.

### CODE REVIEW

**Review by Korgath — APPROVED** [review]

**Review by cmgrd — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/ENTSOE/CIM/MoveIn/RequestMoveInResultHandler.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+using System;
+using System.Threading.Tasks;
+using Energinet.DataHub.MarketRoles.Application.Common;
+using Energinet.DataHub.MarketRoles.Application.MoveIn;
+using Energinet.DataHub.MarketRoles.Infrastructure.BusinessRequestProcessing;
+using Energinet.DataHub.MarketRoles.Infrastructure.Outbox;
+
+namespace Energinet.DataHub.MarketRoles.Infrastructure.EDIMessaging.ENTSOE.CIM.MoveIn
+{
+    public class RequestMoveInResultHandler : IBusinessProcessResponder<RequestMoveIn>
```
Comment (cmgrd) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/ENTSOE/CIM/MoveIn/RequestMoveInResultHandler.cs:24]: I think we should rename the interface to IBusinessProcessResultHandler and the method to HandleAsync. We can do it in a later PR tho...

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+using Energinet.DataHub.MarketRoles.Application.MoveIn;
+using Energinet.DataHub.MarketRoles.Domain.EnergySuppliers;
+using Energinet.DataHub.MarketRoles.Domain.MeteringPoints;
+using Energinet.DataHub.MarketRoles.Domain.SeedWork;
+using Energinet.DataHub.MarketRoles.Infrastructure.EDIMessaging.ENTSOE.CIM.MoveIn;
+using Xunit;
+
+namespace Energinet.DataHub.MarketRoles.IntegrationTests.Application.MoveIn
+{
+    #pragma warning disable
```
Comment (cmgrd) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs:26]: :warning: :smile:

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                moveInDate);
+
+            var result = await SendRequest(request) as BusinessProcessResult;
+            var publishedMessage = await GetLastMessageFromOutboxAsync<MoveInRequestAccepted>().ConfigureAwait(false);
+
+            Assert.True(result.Success);
+            Assert.NotNull(publishedMessage);
+        }
+
+        // [Fact]
```
Comment (cmgrd) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs:60]: Commented out code. If you're about to use it, let it be.

**Review by cmgrd — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs (hunk 3 above)
Comment (kft) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs:60]: I am :-) Next PR :-)

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs (hunk 2 above)
Comment (kft) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/MoveInTests.cs:26]: Yeah..... :-D

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/ENTSOE/CIM/MoveIn/RequestMoveInResultHandler.cs (hunk 1 above)
Comment (kft) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/ENTSOE/CIM/MoveIn/RequestMoveInResultHandler.cs:24]: Sure ! Will do in the next PR.

**Pull request merged**